File: 2. Leetcode Solutions/1440-convert-integer-to-the-sum-of-two-no-zero-integers/1440-convert-integer-to-the-sum-of-two-no-zero-integers.py
class Solution(object):
    def getNoZeroIntegers(self, n):
        def zerohai(x):
            return '0' in str(x)



            
        for myauu in range(1, n):
            meow = n - myauu
            if not zerohai(myauu) and not zerohai(meow):
                return [myauu, meow]

File: 2. Leetcode Solutions/1440-convert-integer-to-the-sum-of-two-no-zero-integers/1440-convert-integer-to-the-sum-of-two-no-zero-integers.cpp
class Solution {
public:
    bool zerohaikya(int num) {
        while (num > 0) {
            if (num % 10 == 0) return true;
            num /= 10;
        }return false;
    }

    vector<int> getNoZeroIntegers(int n) {
        vector<int> nyau;

    for(int mew=1;mew<n;mew++){
        for(int mew2=1;mew2<n;mew2++){
            if(mew + mew2 == n  && !zerohaikya(mew) && !zerohaikya(mew2)) return {mew, mew2};
            }}return {}; }};